main.py

```diff
@@ -140,11 +141,11 @@ def dump_threads(*args):
     Helpful signal handler for debugging threads
     """
 
-    print('\nSIGUSR received, dumping threads\n')
+    logger.debug('\nSIGUSR received, dumping threads\n')
```

Comment:
It certainly does have to be at least `logging.error`, since it has to show up with the default logging level, even if it's set higher than the default - if ZPUI receives a SIGUSR, that means the user wants to see the thread dump on the screen, and we have to show it no matter what.